libs/hdf-converters/src/converters-from-hdf/asff/transformers.ts

```diff
@@ -487,7 +509,22 @@ function createProfileInfoFindingFields(
       }
     });
   });
+  const passThroughObj = _.get(hdf, 'passthrough');
+  if (_.isObject(passThroughObj)) {
+    typesArr = typesArr.concat(
```

Comment:
can do `typesArr.push` here and use the spread operator on the array returned from `pushsplitstring` so that the individual items are added to typesarr

Reply:
done